crates/primitives-traits/src/block/recovered.rs

```diff
@@ -283,6 +283,15 @@ impl<B: Block> RecoveredBlock<B> {
         (self.block.into_block(), self.senders)
     }
 
+    /// Returns the `Recovered<&T>` transaction at the given index.
+    pub fn get_recovered_transaction(
```

Comment:
The common rust lib convention is to leave off the `get_` prefix, because it's redundant

Reply:
I do like it better without the get_ prefix
removing